### PR TITLE
Update dependency-monkey endpoint schema

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

- No

## This Pull Request implements

- Update the schema for the `/dependency-monkey/python/` endpoint to match the requirements of the amun-api `/inspect` endpoint schema
- Update base image used to thoth-s2i recent image and python version to 3.8
- Remove `CutPreReleasesSieve` pipeline unit

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -1120,7 +1124,7 @@ paths:
                     base:
                       type: string
                       description: Base image to build stack upon.
-                      example: quay.io/thoth-station/s2i-thoth-ubi8-py36
+                      example: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.32.3
```
Comment (harshad16) [inline openapi/openapi.yaml:1127]:
```suggestion
                      example: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
```

lets keep this latest, so we wont have to update this schema with each release

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just one minor suggestion

File: openapi/openapi.yaml (hunk 1 above)
Comment (mayaCostantini) [reply openapi/openapi.yaml]: Added in https://github.com/thoth-station/management-api/pull/868/commits/bc244f3dfd741b07ed0e6be47796d36246e706fc, thanks!

**Review by fridex — APPROVED** [review]
perfect 💯

**Pull request merged**